docs/GraFx-Studio/guides/template-variables/date/index.md

```diff
@@ -63,6 +63,53 @@ Will work for patterns which output words such as `MMM`, `MMMM`, `cc` and `ccc`.
 
 ![screenshot-full](date4.png)
 
+### Start & End Date
+
+Setting a start and / or end date, will limit the choice for end users, to pick a date in that range.
+
+A start and end date can be
+
+- None
+- Specific Date
+- Relative Date
+
+![screenshot](date5.png)
+
+#### Specific Date
+
+The possible earliest date, or possible latest date can be made specific. E.g. Headquarters defined that a campaign can be valid within a larger period, but local marketeers can choose the exact start- and end date.
+
+#### Relative Date
+
+Setting a number relative to the current date (current = when you open the document), enables a template designer to set a date prior to the current, or x days after the current date.
+
+As an example, setting 0 in the "Start" date (Type Relative Date), prevents the end user to pick a date in the past.
```

Comment:
setting 0 in the "Start date"

The property is "Start date", not "Start"

Reply:
check